apps/asap-server/src/utils/logger.ts

```diff
@@ -1,4 +1,14 @@
-import debug from 'debug';
+import pino from 'pino-lambda';
+import noir from 'pino-noir';
+import { logLevel } from '../config';
+
+export const redaction = noir(['req.headers.authorization'], '*');
+
+const logger = pino({
+  level: logLevel,
+  formatters: {
+    level: (label: string) => ({ level: label }),
+  },
```

Comment:
```suggestion
  },
  redaction: noir(['req.headers.authorization'], '*'),
```

Is there a reason we can just set redaction by default?

Reply:
it doesnt work if you add it to the logger directly, i have tried and it looks like they actually recommend adding it to express-pino which is pino-http equivalent